lesson07/index.js

```diff
@@ -0,0 +1,45 @@
+const imageArray = [
+    {
+        img: "1.png", 
+        alt:"画像1", 
+        text: "ブックマーク"
+    }, 
+    {
+        img: "2.png", 
+        alt:"画像2", 
+        text: "メッセージ"
+    }
+];
+
+const ul = document.getElementById('js-list');
+const loading = document.getElementById('js-loading');
+const fragment = document.createDocumentFragment();
```

Comment:
move to 

```js
}).then((val) => {
    // here
    for (let i = 0; i < val.length; i++) {
```

because i want to define it near where I use it

Reply:
Thank you for review!
Sorry for the confusion with the other review.
I moved the variables closer together.
[754f3c5](https://github.com/AyakaOnozuka131/JavaScript-lessons/pull/7/commits/754f3c56d7bc346738cec6500a9f3c26e84e7614)